diff -urN liferea-1.5.15/configure.ac liferea-1.5.15.new/configure.ac
--- liferea-1.5.15/configure.ac	2009-03-29 11:09:45.000000000 +0200
+++ liferea-1.5.15.new/configure.ac	2009-03-31 00:36:49.132029106 +0200
@@ -92,7 +92,7 @@
 
 if test "x$enable_lua" = "xyes"; then
    PKG_CHECK_MODULES([LUA], [lua],, [AC_MSG_NOTICE([No lua.pc present, is there a lua5.1.pc?])
-     PKG_CHECK_MODULES([LUA], [lua5.1],, [AC_MSG_NOTICE([Are there lua50.pc and lualib50.pc?])
+     PKG_CHECK_MODULES([LUA], [lua51],, [AC_MSG_NOTICE([Are there lua50.pc and lualib50.pc?])
        PKG_CHECK_MODULES([LUA], [lua50 lualib50],,[enable_lua=no])])])
    
    AC_SUBST(LUA_CFLAGS)
