diff -urN liferea-1.4.17/configure.ac liferea-1.4.17.new/configure.ac
--- liferea-1.4.17/configure.ac	2008-07-22 21:30:09.000000000 +0200
+++ liferea-1.4.17.new/configure.ac	2008-07-22 21:33:08.000000000 +0200
@@ -184,11 +184,6 @@
       AC_SUBST([XULRUNNER_HOME])
       AC_SUBST(XULRUNNER_CFLAGS)
       AC_SUBST(XULRUNNER_LIBS)
-      
-      if test "x$XULRUNNER_HOME" = "x" ; then
-      	AC_MSG_RESULT([Fatal: XulRunner enabled, but XULRUNNER_HOME is empty!])
-      	exit 1
-      fi
 
       MOZILLA_LIB_ROOT=`$PKG_CONFIG --libs-only-L $XULRUNNER_PROVIDER | awk '{print $1}' |  cut -c 3-`
    
diff -urN liferea-1.4.17/src/mozilla/Makefile.am liferea-1.4.17.new/src/mozilla/Makefile.am
--- liferea-1.4.17/src/mozilla/Makefile.am	2007-07-23 19:56:53.000000000 +0200
+++ liferea-1.4.17.new/src/mozilla/Makefile.am	2008-07-22 21:32:47.000000000 +0200
@@ -37,14 +37,22 @@
 
 ## the XulRunner support module
 
+XULRUNNER_EXTRA_CFLAGS = \
+	-I/usr/include/xulrunner \
+	-I/usr/include/xulrunner/dom \
+	-I/usr/include/xulrunner/gtkembedmoz \
+	-I/usr/include/xulrunner/necko \
+	-I/usr/include/xulrunner/webbrwsr \
+	-I/usr/include/xulrunner/xpcom
+
 liblihtmlx_la_SOURCES = \
 	mozembed.c mozembed.h \
 	mozsupport.cpp mozsupport.h \
 	xulrunner.c
 
 liblihtmlx_la_LIBADD = $(PACKAGE_LIBS) $(XULRUNNER_LIBS) $(INTLLIBS)
-liblihtmlx_la_CFLAGS = $(XULRUNNER_CFLAGS) $(PACKAGE_CFLAGS)  -DXULRUNNER_HOME=\""$(XULRUNNER_HOME)\""
-liblihtmlx_la_CXXFLAGS = $(XULRUNNER_CFLAGS) $(PACKAGE_CFLAGS)
+liblihtmlx_la_CFLAGS = $(XULRUNNER_CFLAGS) $(XULRUNNER_EXTRA_CFLAGS) $(PACKAGE_CFLAGS)  -DXULRUNNER_HOME=\""$(XULRUNNER_HOME)\""
+liblihtmlx_la_CXXFLAGS = $(XULRUNNER_CFLAGS) $(XULRUNNER_EXTRA_CFLAGS) $(PACKAGE_CFLAGS)
 
 if PLATFORM_WIN32
 
diff -urN liferea-1.4.17/src/mozilla/xulrunner.c liferea-1.4.17.new/src/mozilla/xulrunner.c
--- liferea-1.4.17/src/mozilla/xulrunner.c	2008-07-01 19:04:55.000000000 +0200
+++ liferea-1.4.17.new/src/mozilla/xulrunner.c	2008-07-22 21:32:47.000000000 +0200
@@ -30,10 +30,12 @@
 
 static void xulrunner_init(void) {
 	
+#ifndef XPCOM_GLUE
 	/* gtk_moz_embed_set_comp_path() must not be called
 	   without a component path for XulRunner. Passing
 	   NULL will crash Liferea. */
 	gtk_moz_embed_set_comp_path(XULRUNNER_HOME);
+#endif
 
 	mozembed_init();	
 }
