diff -ur liferea-1.2-RC3-o/liferea.desktop liferea-1.2-RC3/liferea.desktop
--- liferea-1.2-RC3-o/liferea.desktop	2006-11-25 11:13:15.000000000 -0700
+++ liferea-1.2-RC3/liferea.desktop	2006-11-26 12:07:55.000000000 -0700
@@ -26,5 +26,5 @@
 Terminal=false
 Type=Application
 Encoding=UTF-8
-Categories=Network;News;
+Categories=GTK;Network;News;
 Version=0.9.4
Only in liferea-1.2-RC3: liferea.desktop.orig
