diff -urN liferea-1.6.0-rc6/configure.ac liferea-1.6.0-rc6.new/configure.ac
--- liferea-1.6.0-rc6/configure.ac	2009-06-22 19:31:47.000000000 +0200
+++ liferea-1.6.0-rc6.new/configure.ac	2009-06-30 10:25:41.631646126 +0200
@@ -92,7 +92,7 @@
 
 if test "x$enable_lua" = "xyes"; then
    PKG_CHECK_MODULES([LUA], [lua >= 5.1],, [AC_MSG_NOTICE([No lua.pc >= 5.1 present, is there a lua5.1.pc?])
-     PKG_CHECK_MODULES([LUA], [lua5.1],,[enable_lua=no])])
+     PKG_CHECK_MODULES([LUA], [lua51],,[enable_lua=no])])
    
    AC_SUBST(LUA_CFLAGS)
    AC_SUBST(LUA_LIBS)
