diff -urN liferea-0.9.0b.orig/liferea.desktop liferea-0.9.0b/liferea.desktop
--- liferea-0.9.0b.orig/liferea.desktop	2005-01-20 10:33:00.534927136 +0100
+++ liferea-0.9.0b/liferea.desktop	2005-01-20 10:34:11.591124952 +0100
@@ -4,17 +4,20 @@
 Name[cs]=Liferea Feed Reader
 Name[es]=Liferea, lector de titulares RSS
 Name[fr]=Liferea, lecteur de flux
+Name[pl]=Liferea
 GenericName=Feed Reader
+GenericName[pl]=Czytnik RSS
 Comment=Download and view feeds
 Comment[de]=Anzeigen von News-Feeds
 Comment[cs]=Stahování a čtení článků
 Comment[es]=Lector de titulares RSS Liferea
 Comment[fr]=Télécharge et visualise des flux
+Comment[pl]=Program do pobierania informacji w formacie RSS
 Exec=liferea
 Icon=liferea.png
 StartupNotify=false
 Terminal=false
 Type=Application
 Encoding=UTF-8
-Categories=Application;Network;
+Categories=GTK;Network;News;
 Version=0.9.4
